trace-C19/src/main/webapp/js/script.js

```diff
@@ -1,4 +1,55 @@
 /*
 TODO: Use fetch() to send a POST request at "/outLocation" with a ZIP Code as paramter and 
 you should reive a Json Object with all the locations (lat/long) matching that zip code.
-*/
\ No newline at end of file
+*/
+
+//  const myForm = document.getElementById('zipForm'); 
+
+//  zipForm.addEventListener('submit', function(e) {
+
+//      e.preventDefault();
+
+//      const formData = new FormData(this);
+//      fetch('OutgoingLocationServlet.java', {
+//          method: 'post',
+//          body: formData
+//          .then(function(response) {
+//              return response.text();
+
+//          }) .then(function(text) {
+//              console.log(text);
+
+//          }) .catch(function(error) {
+//             console.error(error);
+//          })
+         
+     
+//      });
+//  })
+
+
+const myForm = document.getElementById('zipForm'); 
+
+myForm.addEventListener('submit', function(e) {
+    e.preventDefault(); 
+
+    const formData = new FormData(this);
+    const searchParams = new URLSearchParams();
+
+    for(const pair of formData) {
+        searchParams.append(pair[0], pair[1]);
+    }
+
+    fetch('http://localhost:8080/outLocation', {
```

Comment:
Instead of `http://localhost:8080/outLocation` I would just put `/outLocation` because when the app goes live it will not be running on the `locaclhost` so that could be a problem.

Reply:
ok, just ommited and sent new one